src/emoji/emoji_map.cr

```diff
@@ -12,879 +12,1495 @@ module Emoji
   # [Emoji Cheat Sheet](http://www.emoji-cheat-sheet.com/).
 
   EMOJI_MAP = {
-    ":+1:"                              => "👍",
```

Comment:
Please run `crystal tool format` and re-commit the file again

Reply:
Done